src/main/java/com/greghaskins/spectrum/Spectrum.java

```diff
@@ -90,7 +90,7 @@ static void compositeSpec(final String context, final com.greghaskins.spectrum.B
    */
   public static void describe(final String context, final com.greghaskins.spectrum.Block block) {
     final Suite suite = getCurrentSuiteBeingDeclared().addSuite(context);
-    suite.applyPreConditions(block);
+    suite.applyPreconditions(block);
```

Comment:
I did think to check that this new method was not yet part of the public API release before renaming it. If I'm wrong then do reject this merge request and I'll revert that change.